libkbfs/block_types.go

```diff
@@ -440,26 +542,63 @@ func (fb *FileBlock) IndirectPtr(i int) (BlockInfo, Offset) {
 	return iptr.BlockInfo, iptr.Off
 }
 
-// OffsetExceedsData implements the Block interface for FileBlock.
-func (fb *FileBlock) OffsetExceedsData(startOff, off Offset) bool {
-	if fb.IsInd {
-		panic("OffsetExceedsData called on an indirect file block")
+// AppendNewIndirectPtr implements the Block interface for FileBlock.
```

Comment:
Some of these say they implement the `Block` interface but they actually implement the `BlockWithPtrs` interface.